code/modules/clothing/head/helmet.dm

```diff
@@ -414,7 +414,7 @@
 
 /obj/item/clothing/head/helmet/monkey_sentience/equipped(mob/user, slot)
 	. = ..()
-	if(slot != ITEM_SLOT_HEAD)
+	if(slot != ITEM_SLOT_HEAD || istype(user, /mob/living/carbon/human/dummy))
```

Comment:
Can you explain why this is necessary?

Reply:
Probably test stuff

Reply:
This code is to prevent the polling code from running when a dummy is given a helmet for the Select Equipment menu.

Reply:
Can you append that as a comment?